guides/stable/layers.md

```diff
@@ -0,0 +1,136 @@
+property of each layer. This property is located inside the layer widget in the
+layers list and is represented by an eye icon.
+
+(layer_opacity)=
+## Layer opacity
+
+All our layers support an opacity slider and `opacity` property that allow you
+to adjust the layer opacity between 0, fully invisible and 1, fully visible. 
+
+```{note}
+* For the [points layer](napari.layers.Points), the opacity value applies
+globally to all the points in the layer, and so you don't need to have any
+points selected for it to have an effect.
+* For the [shapes layer](napari.layers.Shapes), the opacity value applies
+individually to each shape in the layer, and so you must have shapes selected
+for it to have an effect.
+* For the [vectors layer](napari.layers.Vectors), the opacity value applies
+globally to all the vectors in the layer.
+* For the [tracks layer](napari.layers.Tracks), the opacity value applies globally to all the tracks in the layer.
+```
+
+## Blending layers
+
+All our layers support three blending modes: `translucent`, `additive`, and
+`opaque`. These modes determine how the visuals for this layer get mixed with
+the visuals from the other layers.
+
+* An `opaque` layer renders all the other layers below it invisible and will
+fade to black as you decrease its opacity.
+* A `translucent` setting will cause the layer to blend with the layers below
+it if you decrease its opacity but will fully block those layers if its opacity
+is `1`. This is a reasonable default, useful for many applications.
+* An `additive` blending mode will cause the layer to blend with the layers
+below even when it has full opacity. This mode is especially useful for
+visualizing multiple layers at the same time, such as cell biology applications
+where you have multiple different components of a cell labeled in different
+colors.
+
+For example:
+
+![image: blending](../../images/blending.png)
+
+## 3D rendering of images
+
+All our layers can be rendered in both 2D and 3D mode, and one of our viewer
+buttons can toggle between each mode. The number of dimensions sliders will be 2
+or 3 less than the total number of dimensions of the layer, allowing you to
+browse volumetric timeseries data and other high dimensional data. See for
+example these cells undergoing mitosis in this volumetric timeseries:
+
+![image: mitosis](../../images/mitosis.gif)
+
+## Layer interpolation
+
+We support a variety of interpolation modes when viewing 2D slices. In the
+default mode `nearest` each pixel is represented as a small square of specified
+size. As you zoom in you will eventually see each pixel. In other modes
+neighbouring pixels are blended together according to different functions, for
+example `bicubic`, which can lead to smoother looking images. For most
+scientific use-cases `nearest` is recommended because it does not introduce more
+artificial blurring. These modes have no effect when viewing 3D slices.
```

Comment:
```suggestion
scientific use cases, `nearest` is recommended because it displays each data
point, with no mixing of nearby data points. These modes have no effect when viewing 3D slices.
```